spec/features/course/experience_points_record_management_spec.rb

```diff
@@ -7,10 +7,17 @@
   with_tenant(:instance) do
     let(:course) { create(:course) }
     let(:course_student) { create(:course_student, :approved, course: course) }
-    let!(:record) { create(:course_experience_points_record, course_user: course_student) }
-    let!(:inactive_record) do
+    let(:manual_record) { create(:course_experience_points_record, course_user: course_student) }
+    let(:inactive_record) do
       create(:course_experience_points_record, :inactive, course_user: course_student)
     end
+    let(:record) do
+      points_record = create(:course_experience_points_record, course_user: course_student)
+      points_record.actable = points_record
```

Comment:
This is to make it a non-manually-awarded points record. Any concerns if I do this?